qemu/tests/hpt_max_page_size.py

```diff
@@ -0,0 +1,27 @@
+from virttest import error_context
+from virttest.staging.utils_memory import read_from_meminfo
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Test to maximum page size for hash page table guests.
+    Steps:
+    1) There are two options, with men backing file or without men backing
+       file, if without men backing, ignore the step 3.
+    2) System setup hugepages on host.
+    3) Mount this hugepage to /mnt/kvm_hugepage.
+    4) Boot up guest options with different cap-hpt-max-page-size,
+       and there are two modes with kvm or tcg.
+    5) Check the hugepage value is correct inside guest.
+
+    :params test: QEMU test object.
+    :params params: Dictionary with the test parameters.
+    :params env: Dictionary with test environment.
+    """
+    vm = env.get_vm(params["main_vm"])
+    vm.verify_alive()
+    session = vm.wait_for_login()
+
+    if any(read_from_meminfo(_, session) for _ in params['keys'].split()):
```

Comment:
The  keys should be the "unique" output.There's other situation,the output will include the key.

Reply:
The keys include "AnonHugePages" and "ShmemHugePages" , this's the checkpoint of your case.

Reply:
if the code can confirm it is the *only* output,it should be ok.